src/fundus/publishers/de/sz.py

```diff
@@ -14,9 +15,14 @@
 
 class SZParser(ParserProxy):
     class V1(BaseParser):
-        _paragraph_selector = CSSSelector('main [itemprop="articleBody"] > p, ' "main .css-korpch > div > ul > li")
+        _paragraph_selector = XPath(
+            "//div[@itemprop='articleBody']//p[@data-manual='paragraph'" " and not(contains(text(), '© dpa-infocom'))]"
+        )
         _summary_selector = CSSSelector("main [data-manual='teaserText']")
-        _subheadline_selector = CSSSelector("main [itemprop='articleBody'] > h3")
+        _subheadline_selector = XPath(
+            "//div[@itemprop='articleBody']//h3[@data-manual='subheadline']|"
+            "//div[@itemprop='articleBody']//h2[@data-manual='subheadline']"
+        )
```

Comment:
Introducing a new layout is necessary here. If a publisher changes their site layout, it always effects all articles regardless of their publishing time since they are not static but rendered dynamically. The reason we do the extra proxy step is to be able to parse articles which were crawled before the layout changed. The proxy determines the matching parser based on the crawl date and not the publishing date. This is in order to be able to support large HTML archives like CC-NEWS. 

PS: I would stick to a minor version `V1_1` inheriting from `V1` since only the selectors changed.